### ISSUE TITLE
Crash with dump

### ISSUE THREAD

**Issue by jacovandermolen** [trevorsandy/lpub3d#741]
### Subject
Crash running continues page render 

### Environment
LPub3D_x86_64-2.4.7.75.3551_20230905
Windows 10 64b

### Logs
[LPub3DLog.txt](https://github.com/trevorsandy/lpub3d/files/12563400/LPub3DLog.txt)
[LPub3D.dmp.zip](https://github.com/trevorsandy/lpub3d/files/12563410/LPub3D.dmp.zip)

### Steps to reproduce
Open model and go through a few pages.
[31134 - Space Shuttle.txt](https://github.com/trevorsandy/lpub3d/files/12563411/31134.-.Space.Shuttle.txt)

### Actual behaviour
Crash with a model file that gave no problems in previous versions.
Crash seems to be random differs from page to page where crash occurs.

### Workaround
Disable (remove) the 'Parts' view by unchecking Parts in the tabs context menu.

![igxelpicd64 dll_Crash_03](https://github.com/trevorsandy/lpub3d/assets/13388970/b3d92f7e-b942-477e-819b-6a838fad097a)

**Comment by jacovandermolen** [comment]
Then close and restart and then this:
![image](https://github.com/trevorsandy/lpub3d/assets/32261647/71f40251-8dc2-4b4f-a649-e705ebd8c1d2)

**Comment by jacovandermolen** [comment]
Unformatting the model file also crashes LPub.
The file was working fine with previous versions.

**Comment by trevorsandy** [comment]
Thank you for reporting this behaviour.

I took a look at the dmp file and can say I've seen this abnormal end intermittently in development - particularly when building the lcLib (LeoCAD) module.

The abend is triggered by igxelpicd64.dll which is part of the Intel UHD driver and has been reported by several graphics intensive applications to be the source of unexplained crashes - see [Here](https://www.vegascreativesoftware.info/us/forum/fault-mode-crash-igxelpicd64-dll--137659/) or [Here](https://ledstrain.org/d/1407-windows-11-intel-uhd-driver-removed-successfully)

![igxelpicd64 dll_Crash_01](https://github.com/trevorsandy/lpub3d/assets/13388970/e07a3c59-8ad1-44a0-87fd-c5073d604233)

![igxelpicd64 dll_Crash](https://github.com/trevorsandy/lpub3d/assets/13388970/f9e1dbbc-efe2-443d-858d-66cb98a99bdd)

There was no trace within LPub3D to show where the abend is triggered but I suspect it is in the 3DViewer image or display generation.

At this moment, I don't have a solution to resolve this behaviour. I'd am trying to focus on what could be triggering it from within LPub3D. For me the abend is somewhat random and generally at file or page load so this seems consistent with the behaviour you are seeing also.

Cheers,

**Comment by jacovandermolen** [comment]
OK, I see. It is indeed random.
Would it help to disable the 3Dviewer?
The problem does not occur with LPub3D version 2.4.5

I will have to try unformatting again.

My LPub installations are on C:\LEGO\LPub3D...etc

**Comment by trevorsandy** [comment]
> Would it help to disable the 3Dviewer ?

Yes. For me, the abend was quite frequent with the 'Parts' menu enabled and not at all when disabled (unchecked).

![igxelpicd64 dll_Crash_03](https://github.com/trevorsandy/lpub3d/assets/13388970/b3d92f7e-b942-477e-819b-6a838fad097a)

> The problem does not occur with LPub3D version 2.4.5

For me it does. I was able to reproduce the abend when the 'Parts' tab is enabled with v2.4.5.0, v2.4.5.26 and v2.4.5.62.

![igxelpicd64 dll_Crash_04](https://github.com/trevorsandy/lpub3d/assets/13388970/3d4a9992-22f4-44ce-a533-e47d02855398)

Cheers,

**Comment by trevorsandy** [comment]
As suspected, the source of this behaviour was Intel's GPU driver. The bad news is I was able to trigger this behaviour on my latest DevOps build (2.4.7 r186). However, the good news is after updating to Intel's latest graphics driver for my system (09/13/2024), I was able to successfully complete the entire sample document (31134 Space Shuttle) render. It was not necessary to perform the work around of disabling the parts list. In fact, I did not even have to occlude its visibility. Here's a shot of my final render status:

![New_Graphics_Card_Driver](https://github.com/user-attachments/assets/54b2ba11-4100-4196-84f3-65eec575d19a)

New Driver version:
![New_Graphics_Card_Driver_1](https://github.com/user-attachments/assets/498ae4b7-61c4-4357-90fd-d27424f05c28)

Old driver version:
![Old_Graphics_Card_Version](https://github.com/user-attachments/assets/cbd4729a-b885-4b98-9c78-7d317f7fb52c)

Cheers,
